vale/WTD/Just.yml

```diff
@@ -0,0 +1,19 @@
+extends: existence
+message: "Consider removing '%s'"
+ignorecase: true
+scope: text
+level: warning
+tokens:
+  - awesome
```

Comment:
Yea -- I imagine we can apply it to specific directories, for example the guide and the conf websites should probably have different voice/tones.